krun/platform.py

```diff
@@ -513,6 +551,12 @@ def is_virtual(self):
 
         return False
 
+    def get_allowed_dmesg_patterns(self):
+        # PyPy uses write+executable pages, which causes this dmesg entry
+        # in OpenBSD-current (as of around 6.0-beta)
+        return UnixLikePlatform.get_allowed_dmesg_patterns(self) + \
+            [re.compile("pypy\([0-9]+\): mmap W\^X violation$")]
```

Comment:
The more I think of it, the more I think it should be configurable. But I'm keen to push on with instrumentation, so let's address that later.